### PR TITLE
User profile demographics and caching

### PR DESCRIPTION
**Pull request by sharonzhou**
Added education level and income
Added caching of user profile elements: birthday year, ethnicity, gender

### CODE REVIEW

**Comment by dmorina** [comment]
lgtm

File: static/templates/user/profile.html [hunk 1]
```diff
@@ -86,7 +94,7 @@ <h4 style="margin-top:0;margin-bottom:0;">{{ financial_account.balance | currenc
             <div ng-if="vm.edit" layout="column" layout-align="start" class="md-padding" flex>
                 <form>
                     <div layout="column">
-                        <div layout="row" ng-hide="true">
+                        <div layout="row">
                             <md-input-container flex>
                                 <label>First name</label>
                                 <input type="text" ng-model="vm.user.first_name">
```
Comment (shirishgoyal) [inline static/templates/user/profile.html]: Please handle first name and last name for update in backend too

Comment (sharonzhou) [inline static/templates/user/profile.html]: Yes! See next commit

**Comment by sharonzhou** [comment]
Let's see if this comment can re-open the pr

**Pull request closed without merging**